src/Data/Text.hs

```diff
@@ -951,16 +951,17 @@ minimum t = S.minimum (stream t)
 
 -- -----------------------------------------------------------------------------
 -- * Building 'Text's
-
 -- | /O(n)/ 'scanl' is similar to 'foldl', but returns a list of
 -- successive reduced values from the left.
 -- Performs replacement on invalid scalar values.
 --
 -- > scanl f z [x1, x2, ...] == [z, z `f` x1, (z `f` x1) `f` x2, ...]
 --
--- Note that
+-- __Properties__
 --
--- > last (scanl f z xs) == foldl f z xs.
+-- @`head` (`scanl` f z xs) === z@
+-- 
+-- @`last` (`scanl` f z xs) === `foldl` f z xs@
```

Comment:
Any particular reason for `===`? I think outside of `QuickCheck` context `(===)` could be surprising for readers.

Reply:
I just wanted something to indicate observational equivalence for functions (particularly because streams cannot have a reasonable show instance). Do you have an alternative idea for it?

Reply:
I'd follow a convention in `base`, which uses a single `(=)`, e. g., https://hackage.haskell.org/package/base-4.14.0.0/docs/Control-Applicative.html#t:Applicative

Reply:
Ok great, I'll get these fixes to you tonight.